### ISSUE TITLE
Réglages des hyper-paramètres

### ISSUE THREAD

**Issue by lissyx** [Common-Voice/commonvoice-fr#38]
Il s'agit de régler les « bonnes » valeurs aux différents hyper-paramètres du réseau pour obtenir des résultats corrects : nombre d'epoch, learning rate, dropout, lm alpha/beta

 - [x] Nombre d'epoch d'entraînement
 - [x] Learning rate
 - [x] Dropout
 - [x] LM alpha
 - [x] LM beta

**Comment by lissyx** [comment]
J'ai fait pas mal d'essais, et pour le moment, le mieux que j'ai eu c'est : 
```
Epoch 23 |   Training | Elapsed Time: 0:15:28 | Steps: 1120 | Loss: 21.363457
Epoch 23 | Validation | Elapsed Time: 0:00:02 | Steps: 31 | Loss: 9.826485 | Dataset: /mnt/extracted/data/lingualibre/lingua_libre_Q21-fra-French_dev.csv               
Epoch 23 | Validation | Elapsed Time: 0:00:29 | Steps: 91 | Loss: 37.617392 | Dataset: /mnt/extracted/data/trainingspeech/ts_2019-04-11_fr_FR_dev.csv            
Epoch 23 | Validation | Elapsed Time: 0:00:14 | Steps: 65 | Loss: 56.139054 | Dataset: /mnt/extracted/data/cv-fr/clips/dev.csv
I Early stop triggered as (for last 4 steps) validation loss: 39.448354 with standard deviation: 0.281316 and mean: 39.253554
I FINISHED optimization in 6:29:10.898174                                                                                                           
WARNING:tensorflow:From /home/trainer/ds-train-fr/lib/python3.6/site-packages/tensorflow/python/training/saver.py:1266: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:                                                      
Use standard file APIs to check for files with this prefix.
I Restored variables from best validation checkpoint at /mnt/checkpoints/best_dev-25760, step 25760                   
Testing model on /mnt/extracted/data/lingualibre/lingua_libre_Q21-fra-French_test.csv                                
Computing acoustic model predictions | Steps: 62 | Elapsed Time: 0:00:04        
Decoding predictions | 100% (62 of 62) |#| Elapsed Time: 0:00:59 Time:  0:00:59
Test on /mnt/extracted/data/lingualibre/lingua_libre_Q21-fra-French_test.csv - WER: 1.024452, CER: 0.636713, loss: 9.765407
WER: 3.000000, CER: 11.000000, loss: 1.924122                                                                                                                                                                                                                                                                                                                                                                               
 - src: "remilitarisation"                                                       
 - res: "rònis às in"
--------------------------------------------------------------------------------
WER: 3.000000, CER: 6.000000, loss: 4.636756
 - src: "adolescente"
 - res: "à dò lecònte"
--------------------------------------------------------------------------------
WER: 3.000000, CER: 11.000000, loss: 8.666266
 - src: "anatomopathologie"
 - res: "à tòmò àtòlgi"
--------------------------------------------------------------------------------
WER: 3.000000, CER: 4.000000, loss: 16.511086
 - src: "bisontin"
 - res: "is òn in"
--------------------------------------------------------------------------------
WER: 3.000000, CER: 9.000000, loss: 17.462549
 - src: "postdatation"
 - res: "s là tòsien"
--------------------------------------------------------------------------------
WER: 3.000000, CER: 5.000000, loss: 21.161402
 - src: "villuis"
 - res: "i e i"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 8.000000, loss: 0.602511
 - src: "antimonarchiste"
 - res: "endiòn cise"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 7.000000, loss: 0.735558
 - src: "stomatologiste"
 - res: "s tlògise"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 5.000000, loss: 0.830481
 - src: "mongolisme"
 - res: "mòg lse"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 5.000000, loss: 1.360037
 - src: "portâmes"
 - res: "o âe"
--------------------------------------------------------------------------------
Testing model on /mnt/extracted/data/trainingspeech/ts_2019-04-11_fr_FR_test.csv
Computing acoustic model predictions | Steps: 182 | Elapsed Time: 0:00:56
Decoding predictions | 100% (182 of 182) || Elapsed Time: 0:06:58 Time:  0:06:58
Test on /mnt/extracted/data/trainingspeech/ts_2019-04-11_fr_FR_test.csv - WER: 0.993136, CER: 0.666245, loss: 38.370380
--------------------------------------------------------------------------------
WER: 2.000000, CER: 3.000000, loss: 1.150988
 - src: "adieu"
 - res: "à die"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 5.000000, loss: 5.651737
 - src: "arrivez"
 - res: "à ivé"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 3.000000, loss: 8.835285
 - src: "noyée"
 - res: "oi e"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 6.000000, loss: 9.726912
 - src: "jacopo"
 - res: "ce ce"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 3.000000, loss: 10.257656
 - src: "total"
 - res: "s tl"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 5.000000, loss: 11.863169
 - src: "entrons"
 - res: "ee n"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 4.000000, loss: 14.502523
 - src: "baccio"
 - res: "d ci"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 7.000000, loss: 16.017889
 - src: "bellissima"
 - res: "ce ici"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 4.000000, loss: 19.561071
 - src: "samedi"
 - res: "si i"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 6.000000, loss: 21.540262
 - src: "chiarina"
 - res: "s ine"
--------------------------------------------------------------------------------
Testing model on /mnt/extracted/data/cv-fr/clips/test.csv
Computing acoustic model predictions | Steps: 130 | Elapsed Time: 0:00:30
Decoding predictions | 100% (130 of 130) || Elapsed Time: 0:03:23 Time:  0:03:23
Test on /mnt/extracted/data/cv-fr/clips/test.csv - WER: 0.993273, CER: 0.714921, loss: 57.486874                                                                                                                                                                                                                                                                                                                            
-------------------------------------------------------------------------------- 
WER: 2.000000, CER: 5.000000, loss: 4.576832
 - src: "sourires"
 - res: "o ie"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 5.000000, loss: 9.303805
 - src: "anglais"
 - res: "n gi"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 7.000000, loss: 9.741511
 - src: "bonjour"
 - res: "da e"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 9.000000, loss: 12.101145
 - src: "défavorable"
 - res: "di el"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 5.000000, loss: 13.697977
 - src: "courage"
 - res: "o as"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 8.000000, loss: 15.722621
 - src: "lesquelles"
 - res: "di el"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 8.000000, loss: 16.248322
 - src: "malédiction"
 - res: " i ici"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 7.000000, loss: 24.093142
 - src: "lesquels"
 - res: "é le"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 4.000000, loss: 27.016685
 - src: "demainhier"
 - res: "denòin ie"
--------------------------------------------------------------------------------
WER: 2.000000, CER: 12.000000, loss: 92.128860
 - src: "malgré cela"
 - res: "dis inòoi ò ea"
--------------------------------------------------------------------------------
```

L'utilisation réelle montre des résultats proches.

**Comment by lissyx** [comment]
En produisant un modèle ré-utilisant un checkpoint en anglais, bien entraîné, compatible 0.5.0, j'arrive à avoir un début de modèle qui marche un peu correctement.

**Comment by lissyx** [comment]
Avec la v0.3, je commence à avoir quelque chose qui tient la route : 
 - ~15-20 epoch
 - 0.0001 de learning rate
 - 0.3 de dropout
 - lm_alpha testé à 0.65
 - lm_beta testé à 1.45
